proposals/001-ClusterRepresentation.md

```diff
@@ -46,51 +47,60 @@ clusters:
 
 ### Downstream Cluster List
 
-The existing `proxy` configuration will be removed and the existing `clusters` list will be refined to provide a list of `downstream clusters`.  The
+The existing `proxy` configuration will be removed and the existing `clusters` list will be repurposed to provide a list of `downstream clusters`.  The
 downstream cluster list is the list of clusters being presented to the clients. 
 
 Each `downstreamCluster` will define a mapping to an `upstreamCluster`.   The upstream cluster definition provides the bootstrap address of the upstream cluster.
 Kroxylicious will use this to discover the broker's topology.  This is described later.
 
-It is permitted for many downstream clusters to refer to the same upstream cluster. 
+It is permitted for two or more downstream clusters to refer to the same upstream cluster.   This many to one mapping will support multi-tenancy use-cases.
 
 ```yaml
 downstreamClusters:
 - name: mycluster1
   upstream:
-    bootstrap: 123.123.123.123:9092 
+    bootstrap: 123.123.123.123:9092
+    ...
 - name: mycluster2
   upstream:
-    bootstrap: 123.123.123.123:9092 
+    bootstrap: 123.123.123.123:9092
+    ...
 ```
 
-The next section describes the Downstrea Cluster in more details.
+The next section describes the Downstream Cluster in more details.
 
 ### Downstream Cluster 
 
+The downstream cluster provides a name (used for logging and metric labels) and a clusterId (used to identify the cluster to client as a Kafka level).
```

Comment:
I thought we had moved away from talking in terms of `upstream` and `downstream` clusters?